addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1016,6 +1016,25 @@ options.registry.OptionsTab = options.Class.extend({
             hsl.lightness);
         return ColorpickerWidget.convertRgbaToCSSColor(adjustedGrayRGB.red, adjustedGrayRGB.green, adjustedGrayRGB.blue);
     },
+    /**
+     * @override
+     */
+    async _renderCustomXML(uiFragment) {
+        await this._super(...arguments);
+        const rangeExtraSaturation = uiFragment.querySelector('we-range[data-param="gray-extra-saturation"]');
+        if (rangeExtraSaturation) {
+            const defaultSaturationOfGrey = _.range(100, 1000, 100).map(id => {
```

Comment:
Let's not mix US and UK english, use *gray* and not *grey* everywhere in your PR please.
Also as it is a list, name it on plural form so. Here I would name the variable: `baseGraysSaturations`